src/python/pants/goal/run_tracker_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import datetime
+import time
+
+from freezegun import freeze_time
+
+from pants.base.exiter import PANTS_SUCCEEDED_EXIT_CODE
+from pants.goal.run_tracker import RunTracker
+from pants.testutil.option_util import create_options_bootstrapper
+
+
+def test_run_tracker_timing_output() -> None:
+    with freeze_time(datetime.datetime(2020, 1, 1, 12, 0, 0)) as frozen_time:
```

Comment:
you can use it asa decorator. makes thr code more readable and less indented.

Reply:
Can you manually tick time if you use it as a decorator?

Reply:
yes, you can have it pass itself as an arg. see https://github.com/spulec/freezegun#decorator (as_kwarg)